docker_push

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+docker push USER/REPO
```

Comment:
Copy paste error? `USER/REPO` looks like the example string, not what you want to push.

Reply:
Yes, that is correct. What is the repo?

Reply:
that would be mendersoftware/mender-stress-test-client - so far this doesn't exist, but would probably be created on first push